unretire-anyone.lua

```diff
@@ -52,9 +37,13 @@ function showNemesisPrompt(advSetUpScreen)
     if nemesis.figure and not nemesis.flags.ADVENTURER then -- these are already available for unretiring
       local histFig = nemesis.figure
       local histFlags = histFig.flags
-      if (histFig.died_year == -1 or histFlags.ghost) and not histFlags.deity and not histFlags.force then
+      if (histFig.died_year == -1 or histFlags.ghost or options.dead == true) and not histFlags.deity and not histFlags.force then
```

Comment:
No need to check for `== true`. Just "or options.dead" is sufficient.

Reply:
Still know literally nothing about Lua. Thanks!